pkg/model/components/etcd.go

```diff
@@ -23,6 +23,8 @@ import (
 	"k8s.io/kops/upup/pkg/fi/loader"
 )
 
+const DefaultBackupImage = "kopeio/etcd-backup:1.0.20180220"
```

Comment:
Thank-you ... versioning for the win!!

Reply:
You may rue your words tommorow, and the day after, and the day after ;-)